styles/main.css

```diff
@@ -0,0 +1,756 @@
+@import url('https://fonts.googleapis.com/css2?family=Oxygen:wght@300;400;700&display=swap');
+@font-face {
+    font-family: 'et-line';
+    src: url('/resources/fonts/et-line-font/et-line.eot');
+    src: url('/resources/fonts/et-line-font/et-line.eot?#iefix') format('embedded-opentype'), url('/resources/fonts/et-line-font/et-line.woff') format('woff'), url('/resources/fonts/et-line-font/et-line.ttf') format('truetype'), url('/resources/fonts/et-line-font/et-line.svg#et-line') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+:root {
+    --my-grey: #f9f9f9;
+    --my-dark-grey: #ececec;
+    --my-green: #8cc9bb;
+    --font-main-color: #4f4f4f;
+    --font-secundary-color: #858585;
+    --font-title-color: #323232;
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+html, textarea, input, button {
+    font-family: "Oxygen", sans-serif;
+}
+
+body {
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+}
+
+p {
+    font-size: 0.75rem;
+    line-height: 1.8;
+    color: var(--font-main-color);
+}
+
+h3 {
+    color: var(--font-title-color);
+    font-size: 1rem;
+}
+
+.icon {
+    font-size: 1.5rem;
+    font-family: "et-line";
+}
+
+.icon-behance {
+    color: white;
+    background-color: var(--font-title-color);
+    border-radius: 50%;
+}
+
+header {
+    order: 1;
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    background-color: white;
+    box-shadow: 0 1px 5px rgb(240, 240, 240);
+    z-index: 10;
```

Comment:
if you leave the 10 it will mean that there is no way that in the future we can add something over it (as 10 would be the max)

Reply:
Got it